apps/ensapi/src/index.ts

```diff
@@ -61,6 +59,17 @@ app.onError((error, ctx) => {
 // start ENSNode API OpenTelemetry SDK
 sdk.start();
 
+// trigger for warming up the ENSAnalytics aggregated referrer snapshot cache
+const triggerEnsAnalyticsAggregatedReferrerSnapshotCacheWarmup = async () => {
+  logger.info("Warming up ENSAnalytics aggregated referrer snapshot cache...");
+  try {
+    // call the ENSAnalytics Aggregated Referrers endpoint to trigger cache warmup
+    await app.request("/ensanalytics/aggregated-referrers");
```

Comment:
```suggestion
    await app.request("/ensanalytics/referrers");
```

I believe this is updated now. Right?